src/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```

Comment:
add an empty line
also, this file should be located on the same level as pom.xml

Reply:
ooops)0
done